tools/cldr-apps/js/src/esm/cldrInfo.mjs

```diff
@@ -64,6 +60,14 @@ function insertWidget() {
   }
 }
 
+function insertVueElement(containerEl) {
+  const fragment = document.createDocumentFragment();
+  createCldrApp(InfoPanel).mount(fragment); // returns App object "wrapper", currently not saved
+  const vueEl = document.createElement("section");
+  containerEl.appendChild(vueEl);
+  vueEl.replaceWith(fragment);
+}
```

Comment:
could go in cldrVueRouter.js

Reply:
yes eventually, if there were multiple callers, the component (in this case InfoPanel) could be a parameter...